### PR TITLE
C# code

### PR DESCRIPTION
**Pull request by SanyaKirilv**

### CODE REVIEW

File: CourseApp.Tests/ProgramTest.cs [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+        [Fact]
+        public void TestTaskOne1()
+        {
+            var a = 2.0;
+            var b = 3.0;
+            var x = 0.11;
+            var expected = 1.582;
+            Calculate calculateTask = new Calculate();
+            var actual = calculateTask.CalculateTask(a, b, x);
+            Assert.Equal(actual, expected);
```
Comment (alexthvest) [inline CourseApp.Tests/ProgramTest.cs:16]: Please, write first expected value and then actual
According to xUnit docs: `Assert.Equal(expected, actual)`

**Review by alexthvest — CHANGES_REQUESTED** [review]

File: CourseApp.Tests/ProgramTest.cs (hunk 1 above)
Comment (SanyaKirilv) [reply CourseApp.Tests/ProgramTest.cs]: Fixed

File: CourseApp/Program.cs [hunk 2]
```diff
@@ -1,13 +1,75 @@
...
+            var xStart = (double)listTaskOne[0];
+            var xEnd = (double)listTaskOne[1];
+            var xDelta = (double)listTaskOne[2];
+            listTaskOne.Clear();
+            for (double x = xStart; x <= xEnd; x += xDelta)
+            {
+                listTaskOne.Add(x);
+            }
+
+            var listSize = listTaskOne.Count;
```
Comment (alexthvest) [inline CourseApp/Program.cs:30]: Do we need to store count in variable?

File: CourseApp/Program.cs [hunk 3]
```diff
@@ -1,13 +1,75 @@
...
+    using static System.Math;
 
     public class Program
     {
         public static void Main(string[] args)
         {
-            Console.WriteLine($"Hello world");
+            var a = 2.0;
+            var b = 3.0;
+            ArrayList listTaskOne = new ArrayList() { 0.11, 0.36, 0.05 };
```
Comment (alexthvest) [inline CourseApp/Program.cs:13]: Please, do not use ArrayList, use List<T> instead
https://docs.microsoft.com/en-us/dotnet/api/system.collections.arraylist?view=net-5.0

File: CourseApp/Program.cs [hunk 4]
```diff
@@ -1,13 +1,75 @@
...
+            var listSize = listTaskOne.Count;
+            for (int i = 0; i < listSize; i++)
+            {
+                listTaskOne.Add(Calculate(a, b, (double)listTaskOne[i]));
+            }
+        }
+
+        public static void FillTaskTwoArrayList(double a, double b, ArrayList listTaskTwo)
+        {
+            var listSize = listTaskTwo.Count;
```
Comment (alexthvest) [inline CourseApp/Program.cs:39]: Same here

**Review by alexthvest — CHANGES_REQUESTED** [review]

File: CourseApp/Program.cs (hunk 2 above)
Comment (SanyaKirilv) [reply CourseApp/Program.cs:30]: Yes, because the Array List grows like the count.

File: CourseApp/Program.cs (hunk 3 above)
Comment (SanyaKirilv) [reply CourseApp/Program.cs]: Fixed

File: CourseApp/CalculateTaskFolder/CalculateTasks.cs [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+            var listSize = listTaskTwo.Count;
+            for (int i = 0; i < listSize; i++)
+            {
+                listTaskTwo.Add(Calculate(a, b, (double)listTaskTwo[i]));
+            }
+        }
+
+        private void ConsoleOutput(double a, double b, List<double> listTaskOne, List<double> listTaskTwo)
+        {
+            string stringOutput = null;
```
Comment (alexthvest) [inline CourseApp/CalculateTaskFolder/CalculateTasks.cs:61]: You may read about `StringBuilder` 🔕

File: CourseApp/ClassTaskFolder/FileCreator.cs [hunk 6]
```diff
@@ -0,0 +1,180 @@
...
+            Weight = weight;
+            WeightModificator = weightModificator;
+        }
+
+        public FileCreator()
+        : base()
+        {
+            RandomExtension();
+            RandomWeight();
+            RandomWeightModificator();
```
Comment (alexthvest) [inline CourseApp/ClassTaskFolder/FileCreator.cs:25]: Can we move random values generators outside this class? It's bad when the class does a lot of things.

Comment (SanyaKirilv) [reply CourseApp/ClassTaskFolder/FileCreator.cs]: Fixed

File: CourseApp.Tests/TestForCalculateTasks.cs [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+        [InlineData(0.16, 1.592)]
+        [InlineData(0.21, 1.606)]
+        [InlineData(0.26, 1.621)]
+        [InlineData(0.31, 1.637)]
+        [InlineData(0.36, 1.654)]
+        [InlineData(0.08, 1.577)]
+        [InlineData(0.026, 1.571)]
+        [InlineData(0.35, 1.651)]
+        [InlineData(0.41, 1.671)]
+        [InlineData(0.51, 1.701)]
```
Comment (jskonst) [inline CourseApp.Tests/TestForCalculateTasks.cs:18]: Нам прям нужно так много позитивных тестов? Лучше просто проверить критические точки функции

File: CourseApp/CalculateTaskFolder/CalculateTasks.cs [hunk 8]
```diff
@@ -0,0 +1,85 @@
...
+            var sin = Asin(Pow(item, a));
+            var cos = Acos(Pow(item, b));
+            return Round(sin + cos, 3);
+        }
+
+        private void FillTaskOneList(double a, double b, List<double> listTaskOne)
+        {
+            var xStart = listTaskOne[0];
+            var xEnd = listTaskOne[1];
+            var xDelta = listTaskOne[2];
```
Comment (jskonst) [inline CourseApp/CalculateTaskFolder/CalculateTasks.cs:36]: Лучше передать 3 параметра - xStart, dX, xEnd...

File: CourseApp/CalculateTaskFolder/CalculateTasks.cs [hunk 9]
```diff
@@ -0,0 +1,85 @@
...
+            var listTaskOne = new List<double>() { 0.11, 0.36, 0.05 };
+            var listTaskTwo = new List<double>() { 0.08, 0.026, 0.35, 0.41, 0.51 };
+            TaskCalculate(a, b, listTaskOne, listTaskTwo);
+        }
+
+        public double Calculate(double a, double b, double item)
+        {
+            var sin = Asin(Pow(item, a));
+            var cos = Acos(Pow(item, b));
+            return Round(sin + cos, 3);
```
Comment (jskonst) [inline CourseApp/CalculateTaskFolder/CalculateTasks.cs:29]: Зачем нам нужно округлять?

File: CourseApp/CalculateTaskFolder/CalculateTasks.cs [hunk 10]
```diff
@@ -0,0 +1,85 @@
...
+        }
+
+        private void FillTaskOneList(double a, double b, List<double> listTaskOne)
```
Comment (jskonst) [inline CourseApp/CalculateTaskFolder/CalculateTasks.cs:32]: что будет, если я передам только 2 элемента в списке?

File: CourseApp/CalculateTaskFolder/CalculateTasks.cs (hunk 8 above)
Comment (jskonst) [reply CourseApp/CalculateTaskFolder/CalculateTasks.cs:36]: Бегите глупцы :)

File: CourseApp/CalculateTaskFolder/CalculateTasks.cs [hunk 11]
```diff
@@ -0,0 +1,85 @@
...
+            }
+        }
+
+        private void ConsoleOutput(double a, double b, List<double> listTaskOne, List<double> listTaskTwo)
+        {
+            string stringOutput = string.Empty;
+            Console.WriteLine($"Variables: a = {a}; b = {b}");
+            for (int i = 0; i < (listTaskOne.Count / 2); i++)
+            {
+                stringOutput += $"{i + 1}) x = {listTaskOne[i]}  y = {listTaskOne[(listTaskOne.Count / 2) + i]}  ";
```
Comment (jskonst) [inline CourseApp/CalculateTaskFolder/CalculateTasks.cs:65]: форматированный вывод умеет округлять в представлении

File: CourseApp/CalculateTaskFolder/CalculateTasks.cs (hunk 10 above)
Comment (jskonst) [reply CourseApp/CalculateTaskFolder/CalculateTasks.cs:32]: Лучше если метод вернет список (или кортеж списков)

**Review by jskonst — CHANGES_REQUESTED** [review]

**Comment by jskonst** [comment]
В текстах коммитов старайтесь коротко описывать что именно сделано

File: CourseApp.Tests/TestForCalculateTasks.cs (hunk 7 above)
Comment (SanyaKirilv) [reply CourseApp.Tests/TestForCalculateTasks.cs]: Done

File: CourseApp/CalculateTaskFolder/CalculateTasks.cs (hunk 10 above)
Comment (SanyaKirilv) [reply CourseApp/CalculateTaskFolder/CalculateTasks.cs]: Fixed

File: CourseApp/CalculateTaskFolder/CalculateTasks.cs (hunk 8 above)
Comment (SanyaKirilv) [reply CourseApp/CalculateTaskFolder/CalculateTasks.cs]: Done

File: CourseApp/CalculateTaskFolder/CalculateTasks.cs [hunk 12]
```diff
@@ -0,0 +1,87 @@
...
+
+    public class CalculateTasks
+    {
+        public CalculateTasks(double a, double b)
+        {
+            var xStart = 0.11;
+            var xEnd = 0.36;
+            var xDelta = 0.05;
+            var listTaskOne = new List<double>() { };
+            var listTaskTwo = new List<double>() { 0.08, 0.026, 0.35, 0.41, 0.51 };
```
Comment (jskonst) [inline CourseApp/CalculateTaskFolder/CalculateTasks.cs:15]: Allow TaskCalculate decide, create and return list of values

File: CourseApp/CalculateTaskFolder/CalculateTasks.cs [hunk 13]
```diff
@@ -0,0 +1,87 @@
...
+        public CalculateTasks()
+        {
+            var a = 3.0;
+            var b = 2.0;
+            var xStart = 0.11;
+            var xEnd = 0.36;
+            var xDelta = 0.05;
+            var listTaskOne = new List<double>() { };
+            var listTaskTwo = new List<double>() { 0.08, 0.026, 0.35, 0.41, 0.51 };
+            TaskCalculate(a, b, xStart, xEnd, xDelta, listTaskOne, listTaskTwo);
```
Comment (jskonst) [inline CourseApp/CalculateTaskFolder/CalculateTasks.cs:28]: simply call CalculateTask(a,b) instead of code duplicate

File: CourseApp/CalculateTaskFolder/CalculateTasks.cs [hunk 14]
```diff
@@ -0,0 +1,87 @@
+namespace CourseApp
+{
+    using System;
+    using System.Collections.Generic;
+    using static System.Math;
+
+    public class CalculateTasks
+    {
+        public CalculateTasks(double a, double b)
```
Comment (jskonst) [inline CourseApp/CalculateTaskFolder/CalculateTasks.cs:9]: we could use default values for a, b

File: CourseApp/CalculateTaskFolder/CalculateTasks.cs [hunk 15]
```diff
@@ -0,0 +1,87 @@
...
+            var sin = Asin(Pow(item, a));
+            var cos = Acos(Pow(item, b));
+            return sin + cos;
+        }
+
+        private void FillTaskOneList(double a, double b, double xStart, double xEnd, double xDelta, List<double> listTaskOne)
+        {
+            for (double x = xStart; x <= xEnd; x += xDelta)
+            {
+                listTaskOne.Add(x);
```
Comment (jskonst) [inline CourseApp/CalculateTaskFolder/CalculateTasks.cs:42]:
```suggestion
               listTaskOne.Add(Calculate(a, b, x));
```

File: CourseApp/CalculateTaskFolder/CalculateTasks.cs [hunk 16]
```diff
@@ -0,0 +1,87 @@
...
+            }
+
+            var listSize = listTaskOne.Count;
+            for (int i = 0; i < listSize; i++)
```
Comment (jskonst) [inline CourseApp/CalculateTaskFolder/CalculateTasks.cs:46]: to iterate through the list, prefer foreach instead of for

File: CourseApp/CalculateTaskFolder/CalculateTasks.cs [hunk 17]
```diff
@@ -0,0 +1,87 @@
...
+        }
+
+        public double Calculate(double a, double b, double item)
+        {
+            var sin = Asin(Pow(item, a));
+            var cos = Acos(Pow(item, b));
+            return sin + cos;
+        }
+
+        private void FillTaskOneList(double a, double b, double xStart, double xEnd, double xDelta, List<double> listTaskOne)
```
Comment (jskonst) [inline CourseApp/CalculateTaskFolder/CalculateTasks.cs:38]: Return list of y, or list of tuples (x, y)

File: CourseApp/ClassTaskFolder/Document.cs [hunk 18]
```diff
@@ -0,0 +1,40 @@
+namespace CourseApp
```
Comment (jskonst) [inline CourseApp/ClassTaskFolder/Document.cs:1]: Please move this to the separate Pull Request

File: CourseApp/ClassTaskFolder/Document.cs [hunk 19]
```diff
@@ -0,0 +1,40 @@
...
+
+        public Document(string name)
+        {
+            Name = name;
+        }
+
+        public Document()
+        {
+            var generator = new RandomValueGenerator();
+            Name = generator.RandomName();
```
Comment (jskonst) [inline CourseApp/ClassTaskFolder/Document.cs:17]: Bad idea to use random generator in Constructor - you would not be able to control and test it

File: CourseApp/ClassTaskFolder/FileCreator.cs [hunk 20]
```diff
@@ -0,0 +1,88 @@
...
+            WeightModificator = weightModificator;
+        }
+
+        public FileCreator()
+        : base()
+        {
+            var generator = new RandomValueGenerator();
+            Extension = generator.RandomExtension();
+            Weight = generator.RandomWeight();
+            WeightModificator = generator.RandomWeightModificator();
```
Comment (jskonst) [inline CourseApp/ClassTaskFolder/FileCreator.cs:26]: Same here - for random values

File: CourseApp/ClassTaskFolder/RandomValueGenerator.cs [hunk 21]
```diff
@@ -0,0 +1,41 @@
+namespace CourseApp
+{
+    using System;
+    using static System.Math;
+
+    public class RandomValueGenerator
+    {
+        public string RandomName()
```
Comment (jskonst) [inline CourseApp/ClassTaskFolder/RandomValueGenerator.cs:8]: method looks like static

**Review by jskonst — CHANGES_REQUESTED** [review]

File: CourseApp.Tests/TestForCalculateTasks.cs [hunk 22]
```diff
@@ -0,0 +1,26 @@
...
+        [InlineData(0.35, 1.491)]
+        [InlineData(0.41, 1.471)]
+        [InlineData(0.51, 1.441)]
+        public void TestCalculate(double x, double expected)
+        {
+            var a = 3.0;
+            var b = 2.0;
+            var calculateTask = new CalculateTasks();
+            var actual = calculateTask.CalculateValue(a, b, x);
+            Assert.Equal(expected, Round(actual, 3));
```
Comment (alexthvest) [inline CourseApp.Tests/TestForCalculateTasks.cs:23]: Assert.Equal already has overload with round 

```suggestion
            Assert.Equal(expected, actual, 3);
```

File: CourseApp/CalculateTaskFolder/CalculateTasks.cs [hunk 23]
```diff
@@ -0,0 +1,167 @@
...
+            set
+            {
+                CheckValue(value);
+                _bValue = value;
+            }
+        }
+
+        public string Output { get; set; }
+
+        public Tuple<double, double> CalculateValue(double x)
```
Comment (alexthvest) [inline CourseApp/CalculateTaskFolder/CalculateTasks.cs:111]: Use this syntax

```suggestion
        public (double, double) CalculateValue(double x)
```

File: CourseApp/CalculateTaskFolder/CalculateTasks.cs [hunk 24]
```diff
@@ -0,0 +1,167 @@
...
+        {
+            var sin = Asin(Pow(x, AValue));
+            var cos = Acos(Pow(x, BValue));
+            return new Tuple<double, double>(x, sin + cos);
```
Comment (alexthvest) [inline CourseApp/CalculateTaskFolder/CalculateTasks.cs:115]:
```suggestion
            return (x, sin + cos);
```

Comment (alexthvest) [inline CourseApp/CalculateTaskFolder/CalculateTasks.cs:115]: Do we need return `x`? We already have it when we calling this method

File: CourseApp/CalculateTaskFolder/CalculateTasks.cs [hunk 25]
```diff
@@ -0,0 +1,167 @@
...
+        {
+            var sin = Asin(Pow(x, a));
+            var cos = Acos(Pow(x, b));
+            return sin + cos;
+        }
+
+        private void WriteValueInString(double x)
+        {
+            Tuple<double, double> value;
+            value = CalculateValue(x);
```
Comment (alexthvest) [inline CourseApp/CalculateTaskFolder/CalculateTasks.cs:128]:
```suggestion
            var value = CalculateValue(x);
```

File: CourseApp/CalculateTaskFolder/CalculateTasks.cs [hunk 26]
```diff
@@ -0,0 +1,167 @@
...
+            Output += $"x = {value.Item1:f2} y = {value.Item2:f2} | ";
```
Comment (alexthvest) [inline CourseApp/CalculateTaskFolder/CalculateTasks.cs:129]:
```suggestion
            Output += $"x = {x:f2} y = {value.Item2:f2} | ";
```

File: CourseApp/CalculateTaskFolder/CalculateTasks.cs (hunk 25 above)
Comment (alexthvest) [reply CourseApp/CalculateTaskFolder/CalculateTasks.cs:128]: Also you can deconstruct it 
```csharp
var (x, y) = CalculateValue(x);
```

File: CourseApp/ClassTaskFolder/DocumentValue.cs [hunk 27]
```diff
@@ -0,0 +1,103 @@
...
+            {
+                input = Console.ReadLine();
+                isNumber = int.TryParse(input, out int inputValue);
+                if (!isNumber)
+                {
+                    Console.WriteLine("Please enter correct value");
+                }
+            }
+
+            return Convert.ToInt32(input);
```
Comment (alexthvest) [inline CourseApp/ClassTaskFolder/DocumentValue.cs:22]: We already have `inputValue` that has type `int`, we need not to convert input

File: CourseApp/ClassTaskFolder/DocumentValue.cs [hunk 28]
```diff
@@ -0,0 +1,103 @@
+namespace CourseApp.Class
+{
+    using System;
+    using static System.Math;
+
+    public class DocumentValue
+    {
+        public static int IntValue()
```
Comment (alexthvest) [inline CourseApp/ClassTaskFolder/DocumentValue.cs:8]: Method names should be verb

File: CourseApp/ConsoleInputOutput.cs [hunk 29]
```diff
@@ -0,0 +1,129 @@
...
+{
+    using System;
+    using CourseApp.Calculate;
+    using CourseApp.Class;
+
+    public class ConsoleInputOutput
+    {
+        public ConsoleInputOutput()
+        {
+            StartProgram();
```
Comment (alexthvest) [inline CourseApp/ConsoleInputOutput.cs:11]: Please do not run methods in construcor, it's not his work.
Constructor should only check input values if they exist and setup fields and properties

**Review by alexthvest — CHANGES_REQUESTED** [review]

File: CourseApp/ConsoleInputOutput.cs (hunk 29 above)
Comment (SanyaKirilv) [reply CourseApp/ConsoleInputOutput.cs:11]: Fixed

File: CourseApp/ClassTaskFolder/DocumentValue.cs (hunk 27 above)
Comment (SanyaKirilv) [reply CourseApp/ClassTaskFolder/DocumentValue.cs:22]: Fixed

File: CourseApp/CalculateTaskFolder/CalculateTasks.cs (hunk 25 above)
Comment (SanyaKirilv) [reply CourseApp/CalculateTaskFolder/CalculateTasks.cs:128]: Fixed

File: CourseApp/CalculateTaskFolder/CalculateTasks.cs (hunk 23 above)
Comment (SanyaKirilv) [reply CourseApp/CalculateTaskFolder/CalculateTasks.cs:111]: Fixed

File: CourseApp.Tests/TestForCalculateTasks.cs (hunk 22 above)
Comment (SanyaKirilv) [reply CourseApp.Tests/TestForCalculateTasks.cs:23]: Fixed

File: CourseApp/CalculateTaskFolder/CalculateTasks.cs (hunk 24 above)
Comment (SanyaKirilv) [reply CourseApp/CalculateTaskFolder/CalculateTasks.cs]: Fixed

File: CourseApp.Tests/TestForCalculateTasks.cs [hunk 30]
```diff
@@ -0,0 +1,26 @@
...
+        [InlineData(0.36, 1.488)]
+        [InlineData(0.08, 1.565)]
+        [InlineData(0.026, 1.57)]
+        [InlineData(0.35, 1.491)]
+        [InlineData(0.41, 1.471)]
+        [InlineData(0.51, 1.441)]
+        public void TestCalculate(double x, double expected)
+        {
+            var a = 3.0;
+            var b = 2.0;
```
Comment (jskonst) [inline CourseApp.Tests/TestForCalculateTasks.cs:20]: 🔕 a,b may be moved to the inlineData also

File: CourseApp/CalculateTaskFolder/CalculateTasks.cs [hunk 31]
```diff
@@ -0,0 +1,166 @@
...
+            {
+                return _startValue;
+            }
+
+            set
+            {
+                CheckValue(value);
+                _startValue = value;
+            }
+        }
```
Comment (jskonst) [inline CourseApp/CalculateTaskFolder/CalculateTasks.cs:51]: Why do we need porperties?

File: CourseApp/CalculateTaskFolder/CalculateTasks.cs [hunk 32]
```diff
@@ -0,0 +1,166 @@
...
+
+        private void TaskOneList(List<double> listTaskOne)
+        {
+            for (double x = StartValue; x <= EndValue; x += DeltaValue)
+            {
+                listTaskOne.Add(x);
+            }
+        }
+
+        private void StartCalculate(List<double> listTaskOne, List<double> listTaskTwo)
```
Comment (jskonst) [inline CourseApp/CalculateTaskFolder/CalculateTasks.cs:139]: Please split it for 2 methods

**Review by jskonst — CHANGES_REQUESTED** [review]

File: CourseApp/CalculateTaskFolder/CalculateTasks.cs [hunk 33]
```diff
@@ -0,0 +1,166 @@
...
+        {
+            TaskOneList(listTaskOne);
```
Comment (jskonst) [inline CourseApp/CalculateTaskFolder/CalculateTasks.cs:141]: n operations

File: CourseApp/CalculateTaskFolder/CalculateTasks.cs [hunk 34]
```diff
@@ -0,0 +1,166 @@
...
+            Output = "Task One: ";
+            foreach (double x in listTaskOne)
```
Comment (jskonst) [inline CourseApp/CalculateTaskFolder/CalculateTasks.cs:143]: + n operations

File: CourseApp/CalculateTaskFolder/CalculateTasks.cs [hunk 35]
```diff
@@ -0,0 +1,166 @@
...
+            {
+                WriteValueInString(x);
```
Comment (jskonst) [inline CourseApp/CalculateTaskFolder/CalculateTasks.cs:145]: input (str) -> transform to Obj (or numbers) -> operation (numbers, objects) -> Obj -> if need display -> ToString

File: CourseApp/CalculateTaskFolder/CalculateTasks.cs [hunk 36]
```diff
@@ -0,0 +1,166 @@
...
+            }
+
+            Console.WriteLine(Output);
+            Output = "Task Two: ";
+            foreach (double x in listTaskTwo)
+            {
+                WriteValueInString(x);
+            }
```
Comment (jskonst) [inline CourseApp/CalculateTaskFolder/CalculateTasks.cs:153]: Looks like separate method

File: CourseApp/ClassTaskFolder/ClassTask.cs [hunk 37]
```diff
@@ -0,0 +1,20 @@
...
+    using System;
+    using System.Collections.Generic;
+
+    public class ClassTask
+    {
+        public ClassTask(string name, string extension, double weight, string weightModificator)
+        {
+            Document file = new FileCreator(name, extension, weight, weightModificator);
+            Console.WriteLine($"Your file: {file.Display()}");
+        }
```
Comment (jskonst) [inline CourseApp/ClassTaskFolder/ClassTask.cs:12]: Can we move this to the separate Merge Request?

File: CourseApp/ClassTaskFolder/DocumentValue.cs [hunk 38]
```diff
@@ -0,0 +1,105 @@
...
+                    Console.WriteLine($"Value is incorrect. Please enter correct value!");
+                }
+            }
+
+            return inputValue;
+        }
+
+        public string Name()
+        {
+            string[] name = { "isuct", "Funny", "Test", "Car", "Page", "IDE", "Game", "Video", "Music", "Work" };
```
Comment (jskonst) [inline CourseApp/ClassTaskFolder/DocumentValue.cs:46]: let's move this out of method

File: CourseApp/ClassTaskFolder/DocumentValue.cs [hunk 39]
```diff
@@ -0,0 +1,105 @@
...
+
+        private void ConsoleOutput(string[] str)
+        {
+            for (int i = 0; i < str.Length; i++)
+            {
+                Console.WriteLine($"{str[i]} - {i + 1}");
+            }
+        }
+
+        private int CheckIntInput(string[] str)
```
Comment (jskonst) [inline CourseApp/ClassTaskFolder/DocumentValue.cs:85]: why not just pass numbers?
